src/qibo/backends/__init__.py

```diff
@@ -0,0 +1,113 @@
+from qibo import config
+from qibo.config import raise_error, log, warnings, BACKEND_NAME
+from qibo.backends.numpy import NumpyBackend
+from qibo.backends.tensorflow import TensorflowBackend
+
+
+CONSTRUCTED_BACKENDS = {}
+def construct_backend(name):
```

Comment:
For code convention consistency, shall we make this function "private"?

Reply:
I renamed these to `_CONSTRUCTED_BACKENDS` and `_construct_backend`.